### PR TITLE
release: add claimable rewards data to cycle analytics data

### PR DESCRIPTION
**Pull request by Darruma**

### CODE REVIEW

**Review by mitche50 — APPROVED** [review]

File: rewards/calc_rewards.py [hunk 1]
```diff
@@ -189,9 +190,12 @@ def approve_root(
             )
             if chain not in NO_BOOST_CHAINS:
                 upload_boosts(boosts, chain)
+            claimable_rewards_data = get_claimable_rewards_data(chain, end)
```
Comment (btcookies) [inline rewards/calc_rewards.py:193]: if change title of function this could be claimable_rewards_deficits and same in L195

**Review by SHAKOTN — COMMENTED** [review]
Looks good to be approved after addressing @btcookies comments

File: rewards/snapshot/claims_snapshot.py [hunk 2]
```diff
@@ -74,3 +75,16 @@ def claims_snapshot_usd(chain: Network, block: int) -> Tuple[Counter, Counter]:
...
+
+def get_claimable_rewards_data(chain: Network, block: int) -> Dict[str, int]:
+    snapshots = claims_snapshot(chain, block)
+    deficits = {}
+    for token, snapshot in snapshots.items():
+        token_contract = make_token(token, chain)
+        numerator = token_contract.balanceOf(EMISSIONS_CONTRACTS[chain]["BadgerTree"]).call()
+        denominator = math.pow(10, token_contract.decimals().call())
+        tree_balance = numerator / denominator
+        claimable_balance = float(snapshot.total_balance())
```
[the remark this answers is not in the document]
Comment (Darruma) [reply rewards/snapshot/claims_snapshot.py:88]: the total_balance() value is already decimal, we need to do this to for the subtraction to work

**Review by mitche50 — APPROVED** [review]

File: rewards/calc_rewards.py [hunk 3]
```diff
@@ -189,9 +190,14 @@ def approve_root(
             )
             if chain not in NO_BOOST_CHAINS:
                 upload_boosts(boosts, chain)
+            claimable_rewards_data = get_claimable_rewards_deficits(chain, end)
```
Comment (btcookies) [inline rewards/calc_rewards.py:193]: same here

**Review by btcookies — APPROVED** [review]
LGTM

**Pull request merged**